NEWS.md

```diff
@@ -34,6 +34,7 @@ Standard library changes
   otherwise ([#24353]).
 * `IPAddr` subtypes now behave like scalars when used in broadcasting ([#32133]).
 * `clamp` can now handle missing values ([#31066]).
+* `isapprox` (or `≈`) now has a one-argument "curried" method which returns a function, like `==(x)`.
```

Comment:
```suggestion
* `isapprox` (or `≈`) now has a one-argument "curried" method which returns a function, like `==(x)` ([#32305]).
```
Sorry, forgot this in my earlier suggestion for this line. NEWS entries include a reference to an issue or PR.